gallery/src/styles/NxFontSize/NxFontSizePage.tsx

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+const nxFontSizeHtmlExampleCode = require('!!raw-loader!./NxFontSizeHtmlExample.html').default;
+const nxFontSizeScssExampleCode = require('!!raw-loader!./NxFontSizeScssExample.scss').default;
+
+const NxFontSizePage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <p className="nx-p">
+        <code className="nx-code">font-size()</code> is a mixin that can be applied to the
+        {' '}<code className="nx-code">&lt;body&gt;</code> tag of your project to adjust the default RSC font size to
+        better match your project. The default font size in RSC is 16px.
+      </p>
+      <p className="nx-p">
+        The mixin takes a single variable which is the desired font-size with unit. For example:
+        {' '}<code className="nx-code">@include font-size(14px);</code> or
+        {' '}<code className="nx-code">@include font-size(1.1em);</code>.
+      </p>
+      <p className="nx-p">
+        The mixin adjusts the default text size as well as several form elements which have default font size values
+        set by the browser.
+      </p>
+      <p className="nx-p">
+        This mixin is primarily intended for use when RSC is incorporated into existing/legacy projects. If you're
+        using it on a new project then you should confirm with your designer, or the design group that custom font
+        sizes are a design requirement.
+      </p>
+    </GalleryDescriptionTile>
+    <GalleryExampleTile title="Sample HTML"
+                        codeExamples={nxFontSizeHtmlExampleCode}
+                        htmlExample={nxFontSizeHtmlExampleCode}>
```

Comment:
I would have expected an example that actually renders something so the smaller font size can be seen.  You may need to adjust the example to use a `<div>` instead of `<body>`.  If so, add a note to the example explaining that you'd usually use it on `<body>` but for the example it has to be on `<div>`

Reply:
> I would have expected an example that actually renders something so the smaller font size can be seen. You may need to adjust the example to use a `<div>` instead of `<body>`. If so, add a note to the example explaining that you'd usually use it on `<body>` but for the example it has to be on `<div>`

The mixin only works when applied to `<body className="nx-body">` by design since that is where the default font-size is set. I do not think that we should modify the mixin just for the sake of showing an example. Use of this mixin represents a pretty big change to the layout, if the user can't figure out how to use the mixin then I would say that they probably shouldn't be using it at all.

Reply:
> The mixin only works when applied to <body className="nx-body">

Well, it only works when applied to `.nx-body`, at any rate.  Doesn't truly have to be a `<body>` tag.  If you don't actually want it to render anything then the `htmlExample` prop should be removed.  Right now I think it is trying to render that extra `<body>` element, but seems to be failing to actually add it to the DOM probably because it really doesn't want to have two `<body>` tags on the page.

Reply:
> > The mixin only works when applied to
> 
> Well, it only works when applied to `.nx-body`, at any rate. Doesn't truly have to be a `<body>` tag. If you don't actually want this example to render anything then the `htmlExample` prop should be removed. Right now I think it is trying to render that extra `<body>` element, but seems to be failing to actually add it to the DOM probably because it really doesn't want to have two `<body>` tags on the page.

What do you think of this?
https://github.com/sonatype/sonatype-react-shared-components/pull/92/commits/ec790228f3511d08f8ea4055c3ce28a991e2c3b5